jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -54,8 +53,10 @@ def buildExclusions = [
   [ /^smartos17/,                     anyType,     gte(12) ],
   [ /^smartos18/,                     anyType,     lt(12)  ],
 
-  // PPC BE ------------------------------------------------
+  // PPC ---------------------------------------------------
   [ /^ppcbe-ubuntu/,                  anyType,     gte(8)  ],
+  [ /^ppcle-ubuntu/,                  anyType,     gte(13) ],
```

Comment:
This line is new, not just moved, but I can't tell from https://ci.nodejs.org/job/node-test-commit-v8-linux/ what its for. A bit more info in the commit message would be helpful.

Is the version of V8 in master not passing on ppcle-ubuntu? If so, is that an issue we have to look at, or do we not care because we expect 13.x to build on centos7?

Reply:
the tl;dr is we want to test V8 updates on PPC, but one variant is enough. I think the assumption is that for node 13 we'll drop ubuntu14 and prefer CentOS and devtoolset for testing and release.
I'll document this.

Reply:
I don't completely agree with one variant is enough.  Testing on ubuntu as well is good.  Since we'll have to keep it around for earlier releases anyway might be good to keep using 14 until we think about if we can have other machines.  Either way I don't see a rush to change.

Reply:
I am not seeing why we should be testing on different distros unless we have some credible reason to think there are distro (or toolset?) specific problems. V8 shouldn't depend on the distro. Also, isn't this V8 testing for new V8 releases? V8s that aren't going to get ported back to old node.js majors? Its not possible to prove that coverage isn't needed, but there is also a cost in stability, maintainability, and usability as the number of test systems grow. Btw, this is a bit of an aside, but is it impossible to start building the `< 12.x` node.js releases on centos7, even though they used to build on ubuntu1404? I thought centos7+devtoolset gave use more portable binaries, so would be an improvement, and we would decomission the ubuntu1404 machines?